tests/foreach/foreach.t

```diff
@@ -0,0 +1,52 @@
+SETUP PYTHON
+
+$ [ -n "$PYTHON" ] || PYTHON="`which python`"
+$ [ -n "$PYTHONPATH" ] || PYTHONPATH="$TESTDIR/.." && export PYTHONPATH
+
+TEST FOREACH
+
+  $ cp -r $TESTDIR/test_dir_1 test_dir_1
+  $ cd test_dir_1
+  $ git remote add origin file://$TESTDIR/test_dir_1
+  $ cd src/test_dir_2
+  $ git remote add origin file://$TESTDIR/test_dir_1/src/test_dir_2
+  $ cd ../..
+  $ cd src/test_dir_3
+  $ git remote add origin file://$TESTDIR/test_dir_1/src/test_dir_3
+  $ cd ../..
+  $ cd src/test_dir_4
+  $ git remote add origin file://$TESTDIR/test_dir_1/src/test_dir_4
+  $ cd ../..
+  $ quark freeze > /dev/null
+  $ quark foreach 'echo $name $sm_path $displaypath $sha1 $toplevel $rev $version_control'
+  .* (re)
+  .* (re)
+  test_dir_2 src/test_dir_2 src/test_dir_2 bbad4b8fc3c58b39e14956bc81d73b756a08160e /tmp/cramtests-.*/foreach.t/test_dir_1 git (re)
+   (re)
+  test_dir_3 src/test_dir_3 src/test_dir_3 f7e94048b2b2d5a3987ec00ed048fc3fac7a74f0 /tmp/cramtests-.*/foreach.t/test_dir_1 git (re)
+   (re)
+  test_dir_4 src/test_dir_4 src/test_dir_4 13a506afbbae847c8878c04f6f305dff597e3d7d /tmp/cramtests-.*/foreach.t/test_dir_1 git (re)
+   (re)
+
+
+HELPER
+  $ python $TESTDIR/../../bin/quark foreach --help
+  usage: quark foreach [-h] [-q] command [command ...]
+   (re)
+  Evaluates an arbitrary shell command in each submodule, skipping all the svn's
```

Comment:
Is this still true?

Reply:
Actually no, It isn't. I forgot to update the command's description.

Reply:
Changed in https://github.com/comelz/quark/pull/27/commits/50c15a29b6eb9ac8795ac39226b0a7ade8bba6f6